plaster_templates/NewPowerShellModuleProject/test/Module.T.ps1

```diff
@@ -1,9 +1,9 @@
-$ModuleManifestName = '<%=$PLASTER_PARAM_ModuleName%>.psd1'
-$ModuleManifestPath = "$PSScriptRoot\..\$ModuleManifestName"
+[String]$ModuleRoot = Join-Path -Path (Split-Path -Path $PSScriptRoot -Parent) -ChildPath '<%=$PLASTER_PARAM_ModuleName%>'
+Import-Module -Name $ModuleRoot
 
 Describe 'Module Manifest Tests' {
     It 'Passes Test-ModuleManifest' {
-        Test-ModuleManifest -Path $ModuleManifestPath | Should Not BeNullOrEmpty
-        $? | Should Be $true
+        $ManifestPath = Join-Path -Path "$(Split-Path -Path $PSScriptRoot -Parent)" -ChildPath '<%=$PLASTER_PARAM_ModuleName%>/<%=$PLASTER_PARAM_ModuleName%>.psd1'
+        Test-ModuleManifest -Path $ManifestPath | Should -Not -BeNullOrEmpty
```

Comment:
are there meant to be dashes on -Not and -BeNullOrEmpty?

Reply:
pester 5 syntax updates